src/cli/src/lib.rs

```diff
@@ -229,7 +232,22 @@ impl FluvioCmd {
                 produce.process(&fluvio).await?;
             }
             Self::Topic(topic) => {
-                topic.process(out, &fluvio).await?;
+                let result = topic.process(out, &fluvio).await;
+                match &result {
+                    Err(ConsumerError::ClientError(FluvioError::ApiError(api))) => match api {
```

Comment:
Why specific custom handling here instead of just printing out error?

Reply:
That was the point of this whole PR. The behavior we want for `fluvio topic create` is:

- If the topic does not already exist, create it and print `topic "X" created`
- If the topic already exists, do nothing, but print `Topic already exists` and return exit code 1

previously, what would happen is that the `TopicAlreadyExists` error would bubble all the way up and get printed as a color_eyre error with the red error stack. Now, that specific error type is handled in our specified way.

Reply:
(similarly for `fluvio topic delete`)

Reply:
Why not update `eyere` format?

Reply:
I think that the eyre format is good for what it is intended for. I see it as these are two different cases:

1) When trying to create a topic that already exists, this is a "domain error", something that is well-understood and all we need to do is inform the user what happened.
2) When something unexpected goes wrong, this is a "system error", something that we can consider a bug. When this happens, we want to be presented with as much information as possible so that we can understand where the problem originated. This is where eyre's error stack comes in handy: we can get a summary of where the error originated and a general idea of what went wrong.

Reply:
This seems too complex for what we need right now.  Let's just display error and reserve stack based rendering for advanced or debug mode.